compiler/src/dotty/tools/dotc/reporting/ErrorMessageID.scala

```diff
@@ -1,187 +1,190 @@
+  case PkgDuplicateSymbolID // errorNumber: 33
+  case ExistentialTypesNoLongerSupportedID // errorNumber: 34
+  case UnboundWildcardTypeID // errorNumber: 35
+  case DanglingThisInPathID extends ErrorMessageID(isActive = false) // errorNumber: 36
+  case OverridesNothingID // errorNumber: 37
+  case OverridesNothingButNameExistsID // errorNumber: 38
+  case ForwardReferenceExtendsOverDefinitionID // errorNumber: 39
+  case ExpectedTokenButFoundID // errorNumber: 40
+  case MixedLeftAndRightAssociativeOpsID // errorNumber: 41
+  case CantInstantiateAbstractClassOrTraitID // errorNumber: 42
+  case UnreducibleApplicationID // errorNumber: 43
+  case OverloadedOrRecursiveMethodNeedsResultTypeID // errorNumber: 44
+  case RecursiveValueNeedsResultTypeID // errorNumber: 45
+  case CyclicReferenceInvolvingID // errorNumber: 46
+  case CyclicReferenceInvolvingImplicitID // errorNumber: 47
+  case SuperQualMustBeParentID // errorNumber: 48
+  case AmbiguousReferenceID // errorNumber: 49
+  case MethodDoesNotTakeParametersId // errorNumber: 50
+  case AmbiguousOverloadID // errorNumber: 51
+  case ReassignmentToValID // errorNumber: 52
+  case TypeDoesNotTakeParametersID // errorNumber: 53
+  case ParameterizedTypeLacksArgumentsID // errorNumber: 54
+  case VarValParametersMayNotBeCallByNameID // errorNumber: 55
+  case MissingTypeParameterForID // errorNumber: 56
+  case DoesNotConformToBoundID // errorNumber: 57
+  case DoesNotConformToSelfTypeID // errorNumber: 58
+  case DoesNotConformToSelfTypeCantBeInstantiatedID // errorNumber: 59
+  case AbstractMemberMayNotHaveModifierID // errorNumber: 60
+  case TopLevelCantBeImplicitID extends ErrorMessageID(isActive = false) // errorNumber: 61
+  case TypesAndTraitsCantBeImplicitID // errorNumber: 62
+  case OnlyClassesCanBeAbstractID // errorNumber: 63
+  case AbstractOverrideOnlyInTraitsID // errorNumber: 64
+  case TraitsMayNotBeFinalID // errorNumber: 65
+  case NativeMembersMayNotHaveImplementationID // errorNumber: 66
+  case OnlyClassesCanHaveDeclaredButUndefinedMembersID // errorNumber: 67
+  case CannotExtendAnyValID // errorNumber: 68
+  case CannotHaveSameNameAsID // errorNumber: 69
+  case ValueClassesMayNotDefineInnerID // errorNumber: 70
+  case ValueClassesMayNotDefineNonParameterFieldID // errorNumber: 71
+  case ValueClassesMayNotDefineASecondaryConstructorID // errorNumber: 72
+  case ValueClassesMayNotContainInitalizationID // errorNumber: 73
+  case ValueClassesMayNotBeAbstractID // errorNumber: 74
+  case ValueClassesMayNotBeContaintedID // errorNumber: 75
+  case ValueClassesMayNotWrapAnotherValueClassID // errorNumber: 76
+  case ValueClassParameterMayNotBeAVarID // errorNumber: 77
+  case ValueClassNeedsExactlyOneValParamID // errorNumber: 78
+  case OnlyCaseClassOrCaseObjectAllowedID extends ErrorMessageID(isActive = false) // errorNumber: 79
+  case ExpectedTopLevelDefID extends ErrorMessageID(isActive = false) // errorNumber: 80
```

Comment:
I ended up added this in only for the sake of records. I don't think we should rename these, but rather mark them as `active = false` if they are no longer being used. This way they can still be mentioned in the index with their number in case anyone with an old version of the compiler for some reason hits on `E079` and wonders what that is.